roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml

```diff
@@ -13,7 +17,7 @@
       ((thishost.node_ip is undefined) and (thishost.node_name is defined))
     msg: "Invalid delete item. Please define either node_ip only or node_name. In the latter case node_domain is optional."
 
-- name: Ensure that the entry all entries in hosts file are removed with IP {{ thishost.node_ip | d('undefined') }}
+- name: Ensure that all entries with IP {{ thishost.node_ip | d('undefined') }} in /etc/hosts are absent
```

Comment:
Here and other place did throw a lint error. According to lint variables have to be place a the ended of the name line. That is, why I had  this clumsy expression